source/guides/community.rst

```diff
@@ -0,0 +1,81 @@
+=======================================
+Community Engagement Guidelines 
+=======================================
+
+Below is a list of guidelines for maintainers, key contributors and Mattermost, Inc. staff to use when engaging with the broader community in forums, GitHub issues, Twitter, our Mattermost servers and other places of online conversation. 
+
+Best Practices 
+============================================================================
+
+1. Don't offer advice on database modification without a clear warning
+----------------------------------------------------------------------------
+
+Never give DB advice without a warning similar to: `Manual database modification bypass checks in the code and could make a system unrecoverable. It's not recommended and if you do make a change, make sure to back up your DB, and verify changes carefully.`
```

Comment:
Is the text here supposed to be in-line code? If so I think it's two ticks for rst docs. Also this sentence is incorrect (should be modifications bypass or modification bypasses)